client/src/main/java/tds/student/Student.java

```diff
@@ -0,0 +1,67 @@
+package tds.student;
+
+/**
+ * Represents a student taking an exam
+ */
+public class Student {
+    private long id;
+    private String studentId;
+    private String stateCode;
+    private String clientName;
+
+    /**
+     * @return numeric id for the student
+     */
+    public long getId() {
+        return id;
+    }
+
+    /**
+     * @param id numeric id for the student
+     */
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    /**
+     * @return the external SSID in the ART system
+     */
+    public String getStudentId() {
```

Comment:
Oh, I didn't realize it was configurable.  This is the studentid field in the r_studentkeyid table.  A sample value from our test data is `ATHIRDGRADER`.  It really doesn't seem to be used much by the system after login.  I can name it whatever we think it represents.  The external SSID is from a conversation I had with @burzum619